subsys/net/lib/conn_mgr/conn_mgr.h

```diff
@@ -16,14 +16,20 @@
 #define CONN_MGR_IFACE_MAX		CONFIG_NET_IF_MAX_IPV4_COUNT
 #endif
 
+/* Iface internal state flags -- These flags track the internal state of an iface based on the
+ * events it emits.
+ */
 #define CMGR_IF_STATE_UP		BIT(0)
 #define CMGR_IF_STATE_IPV6_SET		BIT(1)
 #define CMGR_IF_STATE_IPV6_DAD_OK	BIT(2)
 #define CMGR_IF_STATE_IPV4_SET		BIT(3)
 
-#define CMGR_IF_STATE_READY		BIT(14)
+/* Event flags -- These flags are used to signal events to the conn_mgr thread */
 #define CMGR_IF_EVT_CHANGED		BIT(15)
 
+/* Virtual state flags -- These flags are updated based on the changes to internal state flags */
+#define CMGR_IF_STATE_READY		BIT(14)
```

Comment:
why reordering it so bit 14th comes after bit 15th?

Reply:
I regrouped the flags by their function, and then sorted the groups by their level of abstraction, but felt a bit odd changing the bit values to maintain order

I've now reassigned the bit values so that they are monotonic